### PR TITLE
Feat: Hyperlink to state page from home

### LINKED ISSUE #1169: Clicking on state navigate to state page
**Is your feature request related to a problem? Please describe.**
It is not that user friendly to enter state names and then navigate to state page, it is kind of frustrating to key in and search for each state.

**Describe the solution you'd like**
Click on any state on the map to navigate to state page

**Describe alternatives you've considered**
- Not applicable -

**Additional context**
Existing search is a addtional feature.

### PR DESCRIPTION
**Pull request by kousiksatish**
**Description of PR**
PR to navigate to state page from table and map

**Type of PR**

- [x] Bugfix

**Relevant Issues**  
Fixes #1169 

**Checklist**

- [x] Compiles and passes lint tests
- [x] Properly formatted
- [x] Tested on desktop
- [x] Tested on phone

**Screenshots**
Screen Shot 2020-04-18 at 8 06 25 PM

### CODE REVIEW

**Comment by kousiksatish** [comment]
Updated 'View state page' to 'Visit state page' to be consistent with search bar results. Please let me know if I need to change.

File: src/components/mapexplorer.js [hunk 1]
```diff
@@ -300,6 +303,18 @@ function MapExplorer({
             Back
           </div>
         ) : null}
+
+        {currentMap.mapType === MAP_TYPES.STATE ? (
+          <Link to={`state/${currentHoveredRegion.statecode}`}>
+            <div
+              className="button state-page-button"
+              onClick={() => switchMapToState('India')}
```
Comment (shuklaayush) [inline src/components/mapexplorer.js]: No need to switch the map

**Review by shuklaayush — CHANGES_REQUESTED** [review]

**Comment by shuklaayush** [comment]
@kousiksatish Nice, but this still needs some work. The hyperlink should only be visible on the home page, and not on the map in the state pages.

File: src/components/mapexplorer.js (hunk 1 above)
Comment (kousiksatish) [reply src/components/mapexplorer.js]: Have updated this code just now.

**Comment by kousiksatish** [comment]
@sudevschiz "The hyperlink should only be visible on the home page, and not on the map in the state pages." - This is also fixed. Please review.

**Review by shuklaayush — APPROVED** [review]

**Review by jeremyphilemon — APPROVED** [review]
Looks good! Thanks for the PR :)

**Pull request merged**